modules/pam_succeed_if/pam_succeed_if.8.xml

```diff
@@ -217,6 +219,24 @@
           <para>(user,host) is not in given netgroup.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term>env:variable is set</term>
+        <listitem>
+          <para>variable has been set to a value.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>env:variable is unset</term>
+        <listitem>
+          <para>variable has not been set to any value.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>env:variable is value</term>
+        <listitem>
+          <para>variable equals value.</para>
+        </listitem>
+      </varlistentry>
```

Comment:
Why do you need this syntax, wouldn't `env:variable = value` be enough?

Reply:
As written in the commit message, for an env variable we may also just want to check if it's only set or unset (!= empty), and since this is something special to env variables only I didn't want to clash with current equality check not change its syntax for a special case.

To allow doing 
env:VARIABLE is unset
env:VARIABLE is set

However this was the thing for which I wasn't fully sure, about which one it was the best way to handle

Reply:
Sorry, I still don't see how `env:VARIABLE is arbitrary_value` is any better than `env:VARIABLE = arbitrary_value`, given that some values have a special meaning.

Reply:
well, it was to have a specific mapping that applies only to env, but I'm fine to change it.

So the idea is to support `= set`? But then how to differentiate it from when the value need to be `"set"`

Reply:
I think it would be fine to have `env:VARIABLE is set|unset` and `env:VARIABLE op value` where `op` is any binary operation like `=`.

Reply:
The question is how to define `env:VARIABLE op value` for various `op` when `env:VARIABLE` is not set.